### DISCUSSION TITLE
I can't find the mistake of connecting with MongoDB Atles

### DISCUSSION THREAD

**Discussion by LuYujin9** [neuefische/questions#568] [Web] [answered]
Hi. 
I have  written code for backend. It works good with local MangoDB.
In order to have preview in Github, I tried to connect it with MongoDB Atles. I did it following the Handout, but it can't work.Maybe I made mistake somewhere, that I can't find out.
Could someone please have a look with me? 

Wenn you would like to have a look at my code. Here ist the PR:
https://github.com/LuYujin9/capstone-project/pull/15/files

**Comment by actyvystom** [comment]
Hi @LuYujin9, 
2 questions first: 
- which error message(s) do you get? Please post them also in this discussion
- where did you place your connection uri env variable? (your `.env.local` file won't work, you will need a `.env` file containing your connection string)

**Reply by actyvystom** [reply, marked as answer]
Problem was a missing database name in the connection string, as a template sample: 
`MONGODB_URI=mongodb+srv://<user>:<password>@cluster0.deezfsg.mongodb.net/<database-name>?retryWrites=true&w=majority`
